modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -295,9 +295,7 @@ public void testGetRelatedCustomObjectEntriesWhenRelationExists()
 			_objectEntry1.getPrimaryKey(), _objectEntry2.getPrimaryKey(),
 			ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
 
-		JSONObject jsonObject = JSONFactoryUtil.createJSONObject(
-			_invoke(
-				Http.Method.GET, _getLocation(objectRelationship.getName())));
+		JSONObject jsonObject = HTTPTestUtil.invoke(null, _getEndpoint(objectRelationship.getName()), Http.Method.GET);
```

Comment:
I think Brian will change `_getEndpoint` for `_getLocation`, that is my bet :joy: :trollface: